NEWS.md

```diff
@@ -1,133 +1,203 @@
-3.35.1
+3.36.1
 ======
 
-## syslog-ng OSE APT repository
+## Highlights
 
-From now on, Ubuntu and Debian packages will be published with every syslog-ng release in the form of an APT repository.
+ * `system()` source: added basic support for reading macOS system logs
 
-We, syslog-ng developers, provide these packages and the APT repository "as is" without warranty of any kind,
-on a best-effort level.
+   The current implementation processes the output of the original macOS syslogd:
+   `/var/log/system.log`.
+   ([#3710](https://github.com/syslog-ng/syslog-ng/pull/3710))
+ * `$(values)` and `$(names)`: these new template functions can be used to
+   query a list of name-value pairs in the current message. The list of name
+   value pairs queried are specified by a value-pairs expression, just like
+   with `$(format-json)`.
 
-Currently, syslog-ng packages are released for the following distribution versions (x86-64):
-  - Debian: bullseye, buster, stretch, sid, testing
-  - Ubuntu: Impish, Focal, Bionic, Xenial
+   Examples:
 
-For instructions on how to install syslog-ng on Debian/Ubuntu distributions, see the
-[README](https://github.com/syslog-ng/syslog-ng/blob/master/README.md#debianubuntu).
+     This expression sets the JSON array `values` to contain the list of SDATA
+     values, while the JSON array `names` would contain the associated names, in
+     the same order.
 
-## Highlights
+     `$(format-json values=list($(values .SDATA.*)) names=list($(names .SDATA.*)))`
+
+   The resulting name-value pairs are always sorted by their key, regardless of
+   the argument order.
+   ([#3911](https://github.com/syslog-ng/syslog-ng/pull/3911))
+ * `tls()` block: added option for restricting TLS 1.3 ciphers
```

Comment:
Optional/opinion: `cipher-suite(tls13())` is not really a big deal. I mean, we obviously need an option to restrict TLS 1.3 ciphers, but it was more of a bug that we forgot to add this in the first place, so I would not list it as a highlight.

The `rename()` rewrite rule or `symlink-as()` however seems to be a good highlight candidate in my opinion.

Reply:
Thanks, fixed both.